src/libraries/System.DirectoryServices/src/Interop/SafeNativeMethods.cs

```diff
@@ -1,34 +1,19 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System.Text;
-using System.Security;
 using System.Runtime.InteropServices;
 
 namespace System.DirectoryServices
 {
     internal static partial class SafeNativeMethods
     {
-        [LibraryImport(global::Interop.Libraries.OleAut32)]
-        public static partial void VariantInit(IntPtr pObject);
-
-        [LibraryImport(global::Interop.Libraries.Activeds)]
-        [return: MarshalAs(UnmanagedType.Bool)]
-        public static partial bool FreeADsMem(IntPtr pVoid);
-
         public const int
             FORMAT_MESSAGE_IGNORE_INSERTS = 0x00000200,
             FORMAT_MESSAGE_FROM_SYSTEM = 0x00001000,
             FORMAT_MESSAGE_ARGUMENT_ARRAY = 0x00002000,
             ERROR_MORE_DATA = 234,
             ERROR_SUCCESS = 0;
 
-        [LibraryImport(global::Interop.Libraries.Activeds, StringMarshalling = StringMarshalling.Utf16)]
-        public static unsafe partial int ADsGetLastError(out int error, char* errorBuffer, int errorBufferLength, char* nameBuffer, int nameBufferLength);
-
-        [LibraryImport(global::Interop.Libraries.Activeds, StringMarshalling = StringMarshalling.Utf16)]
-        public static partial int ADsSetLastError(int error, string? errorString, string? provider);
-
         public sealed class EnumVariant
```

Comment:
It's better to rename this file since it contains `EnumVariant` only now.

Reply:
Should the `EnumVariant` class be moved out of the `SafeNativeMethods` class (and made internal?) too, or leave it as-is?

There are also a couple of consts alongside the `EnumVariant`:
 - three starting `FORMAT_MESSAGE_` appear to be unused, should I delete these?
 - two appear to be only used in combination with `ADsGetLastError` and `ADsSetLastError`, should I move these to an `Interop.ErrorCodes.cs` file under `Common\src\Interop\Windows\Activeds\` perhaps?

Reply:
I'm really not sure for this case, since it's COM interop rather than plain P/invoke.

Reply:
Fair enough, I've left `EnumVariant` nested under `SafeNativeMethods` for now, but have renamed the file, and deleted/moved the consts across 162ce2b60c6e9674e06d141f49da34aec662d425, b100b983705c38ec584398f0c29424ff7e92eb46 and 5aca21963f8b9c1c5af295dbf4f658095f66c8d8 respectively.